lib/cbx_loco/railtie.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails'
+require 'cbx_loco'
+
+module CbxLoco
+  class Railtie < Rails::Railtie
+
+    railtie_name :cbx_loco
+
+    initializer "my_railtie.configure_rails_initialization" do
```

Comment:
Looks to me like my_railtie should be renamed to something else.

Also, there are 2 extraneous empty lines at the beginning of the class block, and at the end of the rake_tasks block.